app/helpers/podcast_metrics_helper.rb

```diff
@@ -0,0 +1,250 @@
+module PodcastMetricsHelper
+  AGENT_TAGS = {
```

Comment:
pulled these tags from `augury/vendors/agents.lock.yml`, and they seem to be match up in the UI from what i can tell?